apps/occurrences/api/views.py

```diff
@@ -19,9 +19,15 @@ class OccurrenceListAPIView(ListAPIView):
     serializer_class = OccurrenceSerializer
 
 
-class OccurrenceAPIView(RetrieveAPIView):
+class OccurrenceSlugAPIView(RetrieveAPIView):
     """API view for a single occurrences."""
 
     queryset = Occurrence.objects.all()
     serializer_class = OccurrenceSerializer
     lookup_field = 'slug'
+
+class OccurrencePkAPIView(RetrieveAPIView):
+    """API view for a single occurrences."""
+
+    queryset = Occurrence.objects.all()
+    serializer_class = OccurrenceSerializer
```

Comment:
Draft state: I still need to add PkAPIViews for other module types, right now we can only request by slug.

Reply:
How about overriding `get_object` so that it looks up by either slug or pk, instead of adding dedicated views for pk lookup?

https://www.django-rest-framework.org/api-guide/generic-views/#get_objectself